src/components/TheFooter.vue

```diff
@@ -52,14 +52,14 @@
               class="footer__version">
               NODE VERSION
               <app-link :to="NODE_RELEASES_URL">
-                v{{ nodeStatus.nodeVersion }}
+                v{{ nodeStatus?.nodeVersion }}
```

Comment:
```suggestion
                {{ formatNullable(nodeStatus.nodeVersion) }}
```

Might be an opportunity to get rid of the extra "v" and also make it use `formatNullable` for this specific case?

Reply:
fixed